doc/ExpertOptions.md

```diff
@@ -1254,33 +1268,23 @@ By default, temporary workspaces hold the results of executed workflows and appl
 }
 ```
 
-This will be passed down through the entire workflow, sub-workflows, and tasks. Workspaces will remain intact for 72 hours. This is a runtime flag, so you will need to run the toplevel workflow with that flag:
+This will be applied to the top-level workflow, sub-workflows, and tasks during compilation.
+
+However, since it is a runtime option, compiling your workflow with this attribute set to true will only guarantee the workspaces of subjobs (that sprawn from the parent job) to remain intact for around 3 days after the analysis. 
+To keep the parent job's workspace, you still need to run the top-level workflow with the runtime flag `--delay-workspace-destruction`.
```

Comment:
```suggestion
To keep the parent jobs' workspaces, you still need to run the top-level workflow with the runtime flag `--delay-workspace-destruction`.
```

Reply:
Are we having only more than one root parent job for each `dx run`? I think the flag only applies to the root that we `dx run` then propagates to all sub jobs/analyses.